puppet/chassis.rb

```diff
@@ -3,6 +3,7 @@
 
 module Chassis
 	@@dir = File.dirname(File.dirname(__FILE__))
+	@@config_dir = @@dir
```

Comment:
This is the new config path variable used to reference the directory that contains the config overrides. Previously it was assumed that all config overrides are inside the Chassis root directory.